backend/pom.xml

```diff
@@ -1,182 +1,187 @@
-            	<systemPropertyVariables>
-    					<logback.configurationFile>${basedir}/logback.xml</logback.configurationFile>
-				</systemPropertyVariables>
-				 <excludes>
-            		<exclude>**/CameraControlTest.java</exclude>
-          		</excludes>
-			</configuration>
-         </plugin>
-         <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>cobertura-maven-plugin</artifactId>
-            <version>2.7</version>
-         </plugin>
-         <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>findbugs-maven-plugin</artifactId>
-            <version>${findbugs.plugin.version}</version>
-            <configuration>
-               <failOnError>false</failOnError>
-            </configuration>
-            <executions>
-               <execution>
-                  <id>verify-findbugs</id>
-                  <phase>process-classes</phase>
-                  <goals>
-                     <goal>check</goal>
-                  </goals>
-               </execution>
-            </executions>
-         </plugin>
-         <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-pmd-plugin</artifactId>
-            <version>${pmd.plugin.version}</version>
-            <configuration>
-               <format>xml</format>
-            </configuration>
-            <executions>
-               <execution>
-                  <id>verify-pmd</id>
-                  <phase>process-classes</phase>
-                  <goals>
-                     <goal>pmd</goal>
-                  </goals>
-               </execution>
-            </executions>
-         </plugin>
-      </plugins>
-   </build>
-   <reporting>
-      <plugins>
-         <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-checkstyle-plugin</artifactId>
-            <version>${checkstyle.plugin.version}</version>
-            <configuration>
-               <configLocation>../checkstyle.xml</configLocation>
-            </configuration>
-         </plugin>
-         <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <version>2.18.1</version>
-         </plugin>
-         <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>cobertura-maven-plugin</artifactId>
-            <version>2.7</version>
-         </plugin>
-         <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>findbugs-maven-plugin</artifactId>
-            <version>${findbugs.plugin.version}</version>
-         </plugin>
-         <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-pmd-plugin</artifactId>
-            <version>${pmd.plugin.version}</version>
-         </plugin>
-      </plugins>
-   </reporting>
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
I don't think it was a good idea to commit a reformat of the pom.xml file in this branch.